examples/src/main/java/org/eclipse/rdf4j/examples/model/Example04LanguageTags.java

```diff
@@ -25,17 +25,15 @@
 
 	public static void main(String[] args) {
 
-		ValueFactory vf = SimpleValueFactory.getInstance();
-
 		// Create a new RDF model containing some information about the painting "The Potato Eaters"
 		ModelBuilder builder = new ModelBuilder();
 		Model model = builder
 				.setNamespace("ex", "http://example.org/")
 				.subject("ex:PotatoEaters")
 				// In English, this painting is called "The Potato Eaters"
-				.add(DC.TITLE, vf.createLiteral("The Potato Eaters", "en"))
+				.add(DC.TITLE, literal("The Potato Eaters", "en"))
 				// In Dutch, it's called "De Aardappeleters"
-				.add(DC.TITLE, vf.createLiteral("De Aardappeleters", "nl"))
+				.add(DC.TITLE, literal("De Aardappeleters", "nl"))
```

Comment:
I know that we like to write our code like this, but it's usually confusing for newbies when there are static imports. Could we use `Values.literal(....)` instead.

Reply:
Yeah fair point.